### PR TITLE
UI

### PR DESCRIPTION
**Pull request by znjs**
Added UI for pages
- Signin
- Signup
- Home
- Explore
- Liked post
- Bookmark

Routes
- "/liked"
- "/bookmark"
- "/explore"
- "/login"
- "/signup"

`Note: Ignore the comment in PostModal and Feed, it is for future purposes`

### CODE REVIEW

File: src/pages/Home/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+              <div className="grid w-10 h-10 bg-base-300 place-items-center">
+                <i className="fa-solid fa-bell"></i>
+              </div>
+            </div>
+          </label>
+          <ul
+            tabIndex="0"
+            className="dropdown-content menu p-2 shadow bg-base-100 rounded-box w-52">
+            <li className="bg-[#454d50] hover:bg-[#2c3133]">
+              <p>Notification 1</p>
```
Comment (shraddha-amber) [inline src/pages/Home/Home.jsx:29]: could've put text directly inside li tag

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                let active = isActive ? "background-clr" : "";
+                return (
+                  " text-lg px-3 py-2 rounded-lg  hover:bg-[#454d50] btn border-0 w-full flex justify-start items-center cursor-pointer my-2 " +
+                  active
+                );
+              }}>
+              <i className="mx-4 fa-solid fa-bookmark"></i> <span> Bookmarks</span>
+            </NavLink>
+            <div className="flex justify-center">
+              {/* <button className="px-3 py-2 rounded-full btn border-0 background-clr hover:bg-[#454d50] my-2 text-base"> */}
```
Comment (shraddha-amber) [inline src/pages/Home/Home.jsx:136]: remove unnecessary comments

**Review by shraddha-amber — APPROVED** [review]
Code is well structured and modular
UI looks good too
Might make the fontSize a little small
Rest is good!

File: src/pages/Home/Home.jsx (hunk 1 above)
Comment (znjs) [reply src/pages/Home/Home.jsx:29]: ok will update that
thank you

File: src/components/PostModal/PostModal.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            type="file"
+            id="imageUpload"
+            name="imageUpload"
+            accept="image/apng, image/avif, image/gif, image/jpeg, image/png, image/svg+xml, image/jpg,image/webp"
+          />
+          <i className="fa-solid fa-image pr-2"></i>
+          Image
+        </button>
+      </label> */}
+              <button className="px-4 py-2 rounded-full  bg-[#11161b] btn border-0 hover:bg-[#11161b70]">
```
Comment (aman-can) [inline src/components/PostModal/PostModal.jsx:39]: **suggestion**:  can remove comments if not in future use

File: src/pages/Home/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+              xmlns="http://www.w3.org/2000/svg"
+              fill="none"
+              viewBox="0 0 24 24"
+              className="inline-block w-5 h-5 stroke-current">
+              <path
+                strokeLinecap="round"
+                strokeLinejoin="round"
+                strokeWidth="2"
+                d="M5 12h.01M12 12h.01M19 12h.01M6 12a1 1 0 11-2 0 1 1 0 012 0zm7 0a1 1 0 11-2 0 1 1 0 012 0zm7 0a1 1 0 11-2 0 1 1 0 012 0z"></path>
+            </svg>
```
Comment (ScoobyDubeyDo) [inline src/pages/Home/Home.jsx:48]: **suggestion**: in place of using SVGs directly in the code, try importing it

File: src/pages/Home/home.css [hunk 5]
```diff
@@ -0,0 +1,9 @@
+.homeContainer {
+    height: calc(100vh - 4.5rem);
+}
+
+.main-container {
+    max-width: 916px;
```
Comment (ScoobyDubeyDo) [inline src/pages/Home/home.css:6]: **suggestion**: could have used relative unit for more responsiveness

**Review by ScoobyDubeyDo — APPROVED** [review]
- The UI looks awesome
- the folder structure was good
- nice pr

File: src/pages/Home/Home.jsx (hunk 4 above)
Comment (znjs) [reply src/pages/Home/Home.jsx:48]: this is taken from library (Daisy UI)

**Pull request merged**